### PR TITLE
Rename unsigned apk

### PR DESCRIPTION
**Pull request by ClaireGizard**
#360

### CODE REVIEW

**Comment by yostyle** [comment]
LGTM. I let giom to approve this PR

**Review by Florian14 — CHANGES_REQUESTED** [review]
Not sure these are the right variants, we should use the `withPinning` variant instead of the `without` one. @giomfo do you confirm?

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        target: [ GplayPreprodWithoutvoipWithoutpinning, FdroidPreprodWithoutvoipWithoutpinning ]
+        target: [ GplayBtchapWithoutvoipWithoutpinning, FdroidBtchapWithoutvoipWithoutpinning ]
```
Comment (Florian14) [inline .github/workflows/build.yml:22]: After discussion IRL, only the gplay variant must be provided **withPinning**

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -46,7 +46,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        target: [ GplayPreprodWithoutvoipWithoutpinning, GplayAgentWithoutvoipWithpinning, FdroidSecureWithoutvoipWithoutpinning, FdroidPreprodWithoutvoipWithoutpinning, FdroidAgentWithoutvoipWithoutpinning ]
+        target: [ GplayBtchapWithoutvoipWithoutpinning, GplayTchapWithoutvoipWithpinning, FdroidBtchapWithoutvoipWithoutpinning, FdroidTchapWithoutvoipWithoutpinning ]
```
Comment (Florian14) [inline .github/workflows/build.yml:49]: Please enable pinning for the GplayBtchap target

File: .github/workflows/integration.yml [hunk 3]
```diff
@@ -20,7 +20,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        target: [ GplayPreprodWithoutvoipWithoutpinning, FdroidPreprodWithoutvoipWithoutpinning ]
+        target: [ GplayBtchapWithoutvoipWithoutpinning, FdroidBtchapWithoutvoipWithoutpinning ]
```
Comment (Florian14) [inline .github/workflows/integration.yml:23]: same here

File: .github/workflows/quality.yml [hunk 4]
```diff
@@ -59,7 +59,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        target: [ GplayPreprodWithoutvoipWithoutpinning, FdroidPreprodWithoutvoipWithoutpinning ]
+        target: [ GplayBtchapWithoutvoipWithoutpinning, FdroidBtchapWithoutvoipWithoutpinning ]
```
Comment (Florian14) [inline .github/workflows/quality.yml:62]: same here

File: .github/workflows/sanity_test.yml [hunk 5]
```diff
@@ -68,7 +68,7 @@ jobs:
             touch emulator.log
             chmod 777 emulator.log
             adb logcat >> emulator.log &
-            ./gradlew $CI_GRADLE_ARG_PROPERTIES -PallWarningsAsErrors=false connectedGplayPreprodWithoutvoipWithoutpinningDebugAndroidTest -Pandroid.testInstrumentationRunnerArguments.class=im.vector.app.ui.UiAllScreensSanityTest || adb pull storage/emulated/0/Pictures/failure_screenshots && exit 1
+            ./gradlew $CI_GRADLE_ARG_PROPERTIES -PallWarningsAsErrors=false connectedGplayBtchapWithoutvoipWithoutpinningDebugAndroidTest -Pandroid.testInstrumentationRunnerArguments.class=im.vector.app.ui.UiAllScreensSanityTest || adb pull storage/emulated/0/Pictures/failure_screenshots && exit 1
```
Comment (Florian14) [inline .github/workflows/sanity_test.yml:71]: probably here as well

File: .github/workflows/tests.yml [hunk 6]
```diff
@@ -26,7 +26,7 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-gradle-
       - name: Run unit tests
-        run: ./gradlew clean TestGplayPreprodWithoutvoipWithoutpinningDebugUnitTest $CI_GRADLE_ARG_PROPERTIES -PallWarningsAsErrors=false --stacktrace
+        run: ./gradlew clean TestGplayBtchapWithoutvoipWithoutpinningDebugUnitTest $CI_GRADLE_ARG_PROPERTIES -PallWarningsAsErrors=false --stacktrace
```
Comment (Florian14) [inline .github/workflows/tests.yml:29]: same here

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -84,16 +84,16 @@ For ktlint to fix some detected errors for you (you still have to check and comm
 #### lint
 
 <pre>
-./gradlew lintGplayPreprodWithoutvoipWithoutpinningRelease
-./gradlew lintFdroidPreprodWithoutvoipWithoutpinningRelease
+./gradlew lintGplayBtchapWithoutvoipWithoutpinningRelease
```
Comment (Florian14) [inline CONTRIBUTING.md:87]: same here

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -84,16 +84,16 @@ For ktlint to fix some detected errors for you (you still have to check and comm
...
+./gradlew lintFdroidBtchapWithoutvoipWithoutpinningRelease
 </pre>
 
 ### Unit tests
 
 Make sure the following commands execute without any error:
 
 <pre>
-./gradlew testGplayPreprodWithoutvoipWithoutpinningReleaseUnitTest
+./gradlew testGplayBtchapWithoutvoipWithoutpinningReleaseUnitTest
```
Comment (Florian14) [inline CONTRIBUTING.md:96]: same here

File: docs/ui-tests.md [hunk 9]
```diff
@@ -60,7 +60,7 @@ Click on the green arrow in front of each test. Clicking on the arrow in front o
 ### From command line
 
 ````shell script
-./gradlew vector:connectedGplayPreprodWithoutvoipWithoutpinningDebugAndroidTest
+./gradlew vector:connectedGplayBtchapWithoutvoipWithoutpinningDebugAndroidTest
```
Comment (Florian14) [inline docs/ui-tests.md:63]: same here

**Review by Florian14 — CHANGES_REQUESTED** [review]

**Review by Florian14 — APPROVED** [review]
SGTM

**Review by giomfo — APPROVED** [review]
👍

**Pull request merged**